scripts/upgrade-test.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# This script will submit a test upgrade proposal, and vote for it.
+# It's assumed that you're running it from the root of this repo.
+# Uses build/provenanced as the executable unless $PROV is set (and exported).
+# Uses build/run/provenanced as PIO_HOME (--home) unless $PIO_HOME is set (and exported).
+
+if [[ -z "$1" ]]; then
+    printf 'Usage: %s <color>\n' "$0"
+    exit 1
+fi
+
+color="$1"
+shift
+
+prov="${PROV:-build/provenanced}"
+
+set -ex
+export PIO_TESTNET=true
+export PIO_HOME="${PIO_HOME:-build/run/provenanced}"
+export PIO_OUTPUT=json
+
+valAddr="$( "$prov" keys list --output json | jq -r '.[] | select( .name == "validator" ) | .address' )"
+curHeight="$( "$prov" status | jq -r '.SyncInfo.latest_block_height' )"
+targetHeight="$(( curHeight + 40 ))"
+
+propRes="$( "$prov" tx gov submit-proposal software-upgrade "$color" \
```

Comment:
Might be helpful to have chain-id as configurable. I've had to update multiple local chains at the same time. Also, might be helpful to have deposit and gas-prices configurable.

Reply:
I've revamped the script so it's all configurable through env vars.

Reply:
Sorry,... forgot to push. Updated for reals now.